mlir/test/Dialect/Linalg/generalize-named-ops.mlir

```diff
@@ -891,3 +975,86 @@ func.func @fill_tensor(%f: f32, %v: vector<2x4xf32>) -> (tensor<f32>, tensor<vec
 
   return %0, %1: tensor<f32>, tensor<vector<2x4xf32>>
 }
+
+// -----
```

Comment:
Why do we need all these generalize tests ?
Seems one would be enough, in conjunction with invalid tests and some (yet missing) new tests to validate the dim inference functionality.